### PR TITLE
Update examples files

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

File: examples/v1alpha1_environment.yaml [hunk 1]
```diff
@@ -6,17 +6,24 @@ metadata:
...
   provider: aws
   auth:
-    keyName: cnt-ci
-    privateKey: "/Users/eduardoa/.ssh/cnt-ci.pem"
+    keyName: HOLODECK_AWS_ACCESS_KEY_ID
+    privateKey: HOLODECK_AWS_SECRET_ACCESS_KEY
   instance:
     type: g4dn.xlarge
-    region: us-west-1
+    region: eu-north-1
```
[the remark this answers is not in the document]
Comment (ArangoGutierrez) [reply examples/v1alpha1_environment.yaml:13]: yes, the aws-sdk pkg does that by default, as well as the key and key_id

File: examples/v1alpha1_environment.yaml [hunk 2]
```diff
@@ -6,17 +6,24 @@ metadata:
...
     ingressIpRanges:
-      - 0.0.0.0/0
+      - 213.179.129.0/26
     image:
       architecture: amd64
-      imageId: ami-0ce2cb35386fc22e9
+      imageId: ami-0fe8bec493a81c7da
```
[the remark this answers is not in the document]
Comment (ArangoGutierrez) [reply examples/v1alpha1_environment.yaml:18]: https://github.com/NVIDIA/holodeck/pull/63

File: examples/v1alpha1_environment.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ArangoGutierrez) [reply examples/v1alpha1_environment.yaml:13]: is to provide an example as complete as posible

File: examples/v1alpha1_environment.yaml [hunk 3]
```diff
@@ -6,17 +6,24 @@ metadata:
...
-    privateKey: "/Users/eduardoa/.ssh/cnt-ci.pem"
+    keyName: HOLODECK_AWS_ACCESS_KEY_ID
+    privateKey: HOLODECK_AWS_SECRET_ACCESS_KEY
   instance:
     type: g4dn.xlarge
-    region: us-west-1
+    region: eu-north-1
     ingressIpRanges:
-      - 0.0.0.0/0
+      - 213.179.129.0/26
```
[the remark this answers is not in the document]
Comment (ArangoGutierrez) [reply examples/v1alpha1_environment.yaml]: Done

File: examples/v1alpha1_environment.yaml [hunk 4]
```diff
@@ -6,17 +6,24 @@ metadata:
...
+    name: containerd
+  nvidiaContainerToolkit:
+    install: true
+  nvidiaDriver:
+    install: true
   kubernetes:
     install: true
     installer: kubeadm
-    version: v1.28.5
+    version: v1.28.5
```
[the remark this answers is not in the document]
Comment (ArangoGutierrez) [reply examples/v1alpha1_environment.yaml]: Done

**Review by elezar — APPROVED** [review]

**Pull request merged**